src/app/pages/LiveRadioPage/index.test.jsx

```diff
@@ -6,34 +6,37 @@ import { matchSnapshotAsync } from '@bbc/psammead-test-helpers';
 import { ServiceContextProvider } from '#contexts/ServiceContext';
 import { RequestContextProvider } from '#contexts/RequestContext';
 import LiveRadioPage from '.';
-import amharicPageData from '#data/amharic/bbc_amharic_radio/liveradio';
+import afriquePageData from './fixtureData/afrique';
+import indonesianPageData from './fixtureData/indonesia';
+import gahuzaPageData from './fixtureData/gahuza';
 import * as analyticsUtils from '#lib/analyticsUtils';
 import { ToggleContextProvider } from '#contexts/ToggleContext';
-import getInitialData from '#app/routes/liveRadio/getInitialData';
 
-const Page = ({ pageData, isAmp = false }) => (
+const Page = ({ pageData, service = 'afrique', isAmp = false }) => (
```

Comment:
Could be wrong but I think because the default of this prop is being set within the `renderPage` function, it doesn't need setting here as well

Reply:
Agreed. I clocked it earlier and then forgot to remove it